README.md

```diff
@@ -834,6 +834,32 @@ func main() {
 }
 ```
 
+### Serving data from reader
```

Comment:
Please add content link in README, thanks!

Reply:
Hello @thinkerou I'm not sure to understand the target of the asked link. This page ?

Reply:
You should add `[Serving data from reader](serving-data-from-reader)` after the line:
![image](https://user-images.githubusercontent.com/4488261/39750567-23b6b2b2-52e8-11e8-9637-38236d8c81e8.png)